hack/release.sh

```diff
@@ -70,6 +70,32 @@ push_all() {
     push_pre_upgrade_checks_image latest
 }
 
+
+
+# Build and push fission-bundle image
+push_multiarch_fission_bundle_image() {
+    TAG=$1 make multiarch-bundle
+}
+
+push_multiarch_fetcher_image() {
+    TAG=$1 make multiarch-fetcher
+}
+
+
+push_builder_image() {
+    TAG=$1 make multiarch-builder
+}
+
+# Push pre-upgrade-checks image
+push_pre_upgrade_checks_image() {
+    TAG=$1 make multiarch-preupgrade
+}
```

Comment:
Do we need these given we have the image-multiarch which covers all images?

Reply:
Might help someone who is just working to test one particular component especially if it's on ARM or something other than AMD.

Reply:
Sure, let it be around!